lib/main.dart

```diff
@@ -197,10 +198,7 @@ Map<String, dynamic>? loadConfigFileFromArgResults(ArgResults argResults, {bool
 }
 
 Map<String, dynamic> loadConfigFile(String path, String? fileOptionResult) {
-  final File file = File(path);
-  final String yamlString = file.readAsStringSync();
-  // ignore: always_specify_types
-  final Map yamlMap = loadYaml(yamlString);
+  final yamlMap = PubspecParser.toMap(path);
 
   if (!(yamlMap['flutter_icons'] is Map)) {
```

Comment:
```suggestion
  if (yamlMap['flutter_icons'] is! Map) {
```

[`prefer_is_not_operator`](https://dart.dev/tools/linter-rules#prefer_is_not_operator)

Reply:
Updated